testcases.py

```diff
@@ -1239,15 +1241,10 @@ def check(self) -> TestResult:
             self._server_trace()._get_direction_filter(Direction.FROM_SERVER) + " quic"
         )
 
-        ports = list(set(getattr(p["udp"], "dstport") for p in tr_server))
-
-        logging.info("Server saw these client ports: %s", ports)
-        if len(ports) <= 1:
-            logging.info("Server saw only a single client port in use; test broken?")
-            return TestResult.FAILED
```

Comment:
Why is this no longer helpful?

Reply:
This is now done below based on the other info extracted, see the `Server saw the client use only a single path; test broken?` error.

Reply:
Oh, yes, thanks for clarifying.